omnigibson/simulator.py

```diff
@@ -872,6 +993,13 @@ def _on_contact(self, contact_headers, contact_data):
                                     obj1, headers[(actor0_obj, actor1_obj)], contact_data
                                 )
 
+        def find_object_in_scenes(self, prim_path):
```

Comment:
nit: maybe more accurate naming along the lines of "get_obj_at_prim_path" or something like that?